redis+sentinel cannot be used as core chart cache

**Expected behavior and actual behavior:**
**Expected:** redis+sentinel can be used for chart cache.
**Actual:** redis+sentinel cannot be used for chart cache

**Steps to reproduce the problem:**
* Run harbor core with `CHART_CACHE_DRIVER=redis_sentinel`
* Observe harbor core log.
```
2020-11-10T15:03:25Z [ERROR] [/chartserver/controller.go:62]: failed to get cache configuration with error: cache driver 'redis_sentinel' is not supported, only support 'memory' and 'redis'
2020-11-10T15:03:25Z [INFO] [/chartserver/controller.go:66]: No cache is enabled for chart caching
```

**Versions:**
Please specify the versions of following systems.

- harbor version: [2.1.1]

**Additional context:**

`redis_sentinel` driver cannot be used because only `redis` or `memory` are allowed:
https://github.com/goharbor/harbor/blob/d32527dfd77a87f8a9913904c0652146e9c32c38/src/chartserver/cache.go#L20-L22
https://github.com/goharbor/harbor/blob/d32527dfd77a87f8a9913904c0652146e9c32c38/src/chartserver/cache.go#L62-L64

--- a/src/chartserver/cache.go
+++ b/src/chartserver/cache.go
@@ -59,11 +59,11 @@ func NewChartCache(config *ChartCacheConfig) *ChartCache {
 		return chartCache
 	}
 
-	if config.DriverType != cacheDriverMem && config.DriverType != cacheDriverRedis {
+	if config.DriverType != cacheDriverMem && config.DriverType != cacheDriverRedis && config.DriverType != cacheDriverRedisSentinel {
 		return chartCache
 	}
 
-	if config.DriverType == cacheDriverRedis {
+	if config.DriverType == cacheDriverRedis || config.DriverType == cacheDriverRedisSentinel {
 		if len(config.Config) == 0 {
 			return chartCache
 		}
@@ -107,7 +107,7 @@ func (chc *ChartCache) PutChart(chart *ChartVersionDetails) {
 		case cacheDriverMem:
 			// Directly put object in
 			err = chc.cache.Put(chart.Metadata.Digest, chart, standardExpireTime)
-		case cacheDriverRedis:
+		case cacheDriverRedis, cacheDriverRedisSentinel:
 			// Marshal to json data before saving
 			var jsonData []byte
 			if jsonData, err = json.Marshal(chart); err == nil {
@@ -193,7 +193,7 @@ func initCacheDriver(cacheConfig *ChartCacheConfig) beego_cache.Cache {
 			redisCache, err := beego_cache.NewCache(cacheDriverRedisSentinel, cacheConfig.Config)
 			if err != nil {
 				// Just logged
-				hlog.Errorf("Failed to initialize redis cache: %s", err)
+				hlog.Errorf("Failed to initialize redis sentinel cache: %s", err)
 
 				if count < maxTry {
 					<-time.After(time.Duration(backoff(count)) * time.Second)
@@ -203,7 +203,7 @@ func initCacheDriver(cacheConfig *ChartCacheConfig) beego_cache.Cache {
 				return nil
 			}
 
-			hlog.Info("Enable redis cache for chart caching")
+			hlog.Info("Enable redis sentinel cache for chart caching")
 			return redisCache
 		}
 	default:

--- a/src/chartserver/handler_utility.go
+++ b/src/chartserver/handler_utility.go
@@ -175,7 +175,7 @@ func (c *Controller) GetChartVersionDetails(namespace, chartName, version string
 		chartDetails.Security.Signature.Provenance = provFilePath
 	} else {
 		// Just log it
-		hlog.Errorf("Failed to get prov file for chart %s with error: %s, got %d bytes", chartV.Name, err.Error(), len(provBytes))
+		hlog.Debugf("Failed to get prov file for chart %s with error: %s, got %d bytes", chartV.Name, err.Error(), len(provBytes))
 	}
 
 	return chartDetails, nil

--- a/src/chartserver/utils.go
+++ b/src/chartserver/utils.go
@@ -130,7 +130,7 @@ func getCacheConfig() (*ChartCacheConfig, error) {
 		return nil, fmt.Errorf("failed to parse redis configurations from '%s' with error: %s", redisCfg, err)
 	}
 	if _, isSet := redisCfg["masterName"]; isSet {
-		driver = "redis_sentinel"
+		driver = cacheDriverRedisSentinel
 	}
 
 	// Convert config map to string

